.github/workflows/linux.yml

```diff
@@ -388,7 +388,7 @@ jobs:
           version: apt-22-java
           execute_install_scripts: true
       - run: .github/setup-linux.sh
-      - run: .github/build.sh piv-sm dist valgrind
+      - run: .github/build.sh piv-sm dist
```

Comment:
I had already somethin similar  queued https://github.com/OpenSC/OpenSC/pull/3497

Reply:
Sorry for a confusion, this line was a mistake and should not be there. Removed.